server/@types/express/index.d.ts

```diff
@@ -13,7 +13,7 @@ declare module 'express-session' {
     slotsList: VisitSlotList
     visitSessionData: VisitSessionData
     selectedEstablishment: Prison
-    visitBlockDate?: string
+    visitBlockDate?: string // format YYYY-MM-DD
```

Comment:
Might be able to be more specific (just found this out though!) 

`type idType = `${number}-${number}-${number}``

Reply:
Hmm... interesting. I didn't know you could do that! Just tried it and it sort of worked - had to replace `string` with `${number}-${number}-${number}` in a few places but it didn't complain. Would probably make sense to define that itself as a type like `SHORT_DATE` I guess. Interesting 🤔!

Reply:
Likewise!